shared/stateutil/state_root_cache_fuzz_test.go

```diff
@@ -6,45 +6,35 @@ import (
 
 	fuzz "github.com/google/gofuzz"
 	ethereum_beacon_p2p_v1 "github.com/prysmaticlabs/prysm/proto/beacon/p2p/v1"
+	"github.com/prysmaticlabs/prysm/shared/params"
 )
 
-func TestStateRootCacheFuzz_100(t *testing.T) {
-	fuzzStateRootCache(t, 0, 100)
+func init() {
+	params.UseMinimalConfig()
 }
 
-func TestStateRootCacheFuzz_1000(t *testing.T) {
-	fuzzStateRootCache(t, 1, 1000)
+func TestStateRootCacheFuzz_10(t *testing.T) {
+	fuzzStateRootCache(t, 0, 10)
 }
 
-func TestStateRootCacheFuzz_10000(t *testing.T) {
-	fuzzStateRootCache(t, 2, 10000)
-}
-
-func TestStateRootCacheFuzz_100000(t *testing.T) {
-	fuzzStateRootCache(t, 3, 100000)
-}
-
-func TestStateRootCacheFuzz_1000000(t *testing.T) {
```

Comment:
Why were the larger cases removed?

Reply:
They take way too long to run now. Now, when doing HTR(state.BlockRoots), we pad the slice to params.BeaconConfig().SlotsPerHistoricalRoot, and the fuzz tests don't really make much use of the cache so much because the inputs change on every iteration, so they take almost 10s to run the 10k fuzz test, let alone the 100k one.